RxSwiftWorkshop/Flights/Model/ServiceType.swift

```diff
@@ -0,0 +1,23 @@
+//
+// Created by Maciej Oczko on 01.05.2017.
+// Copyright (c) 2017 Mobile Academy. All rights reserved.
+//
+
+import Foundation
+
+enum ServiceType {
```

Comment:
Yes, you're right.

I totally forgot about that we have have different String values assigned to case. Silly me.